### DISCUSSION TITLE
Welcome to NOSC-Website Discussions!

### DISCUSSION THREAD

**Discussion by IshaanAdarsh** [NITRR-Open-Source-Community/NOSC-Website#15] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Prakashd20** [comment]
Hey @IshaanAdarsh ,
I'm Prakash Dubey who is currently pursuing his bachelor's in computer science from Gujarat Technological University and having a keen interest in open source programs.

**Reply by IshaanAdarsh** [reply]
Hi @Prakashd20, take a pick of the open issues and start on the basics to solve it. If you have any issues just tag me or @nown1ne, and we will help you out as much as we can

**Reply by Prakashd20** [reply]
@IshaanAdarsh actually I'm new to open source, so would need more of the guidance throughout to work with.

**Reply by IshaanAdarsh** [reply]
I would suggest start with the documentation issues. You would understand the tech stack more and how to navigate the project.
https://github.com/NITRR-Open-Source-Community/NOSC-Website/issues/12
https://github.com/NITRR-Open-Source-Community/NOSC-Website/issues/44

**Comment by hemantgovekar** [comment]
@IshaanAdarsh  and @nown1ne  : I can work on some issues on this website. Do you have any priority issue which you need to work on. Let me know.

**Reply by IshaanAdarsh** [reply]
@hemantgovekar we don't have a priority set for the issues. You can take up anyone you are interested in.
